saleor/graphql/extensions/types.py

```diff
@@ -24,3 +24,25 @@ class Meta:
         model = models.PluginConfiguration
         interfaces = [graphene.relay.Node]
         only_fields = ["name", "description", "active", "configuration"]
+
+    def resolve_configuration(self, _info):
+        plugin = manager.get_extensions_manager().get_plugin(self.name)
+        configuration = plugin.get_plugin_configuration().configuration
+        if plugin.CONFIG_STRUCTURE and configuration:
+            for field in configuration:
+                name = field["name"]
+                value = field["value"]
+                if value is None:
+                    continue
+                field_type = plugin.CONFIG_STRUCTURE.get(name, {}).get("type")
+                if field_type == ConfigurationTypeField.PASSWORD:
+                    field["value"] = "" if value else None
+
+                if field_type == ConfigurationTypeField.SECRET:
```

Comment:
Maybe moving this to some separate function would be a good idea? I'm pretty sure that this will be needed somewhere else :)